modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/edit_sxp_blueprint/EditSXPBlueprintForm.js

```diff
@@ -477,8 +508,8 @@ function EditSXPBlueprintForm({
 			sortConfiguration: sortConfig ? JSON.parse(sortConfig) : {},
 		};
 
-		if (indexConfig) {
-			configuration.indexConfiguration = indexConfig;
+		if (featureFlagLps153813) {
+			configuration.indexConfiguration = indexConfig || {};
```

Comment:
The acceptance criteria says that if company index was being used, it'd default to empty. Was wondering if that meant defining `indexConfiguration` at all? Also would the default value be `{indexName: ''}`? Though I'm not sure if it'd ever reach that part.

Reply:
Yes the default is set to an empty value `{indexName: ''}` here:
https://github.com/liferay-search/liferay-portal/blob/45bdc16ab5af8e730687b135503645b830a338df/modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/edit_sxp_blueprint/EditSXPBlueprintForm.js#L400-L402

Reply:
And when a user selects the "Company Index" selection this is where the empty value is set as well.
https://github.com/liferay-search/liferay-portal/blob/45bdc16ab5af8e730687b135503645b830a338df/modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/edit_sxp_blueprint/configuration_tab/index.js#L58